### PR TITLE
Doc: Add documentation for SFrame.read_csv() nrows_to_infer

### PR DESCRIPTION
**Pull request by tapaswenipathak**
Fixes https://github.com/apple/turicreate/issues/1367.

### CODE REVIEW

File: src/python/turicreate/data_structures/sframe.py [hunk 1]
```diff
@@ -1270,6 +1270,9 @@ def read_csv(cls,
             continue parsing skipping lines which fail to parse correctly.
             A sample of the first 10 encountered bad lines will be printed.
 
+        nrows_to_infer: integer
```
Comment (TobyRoseman) [inline src/python/turicreate/data_structures/sframe.py]: Let's have the ordering of parameters in the docstring match the order of parameters in the function definition.

File: src/python/turicreate/data_structures/sframe.py [hunk 2]
```diff
@@ -1270,6 +1270,9 @@ def read_csv(cls,
...
+            Get the number of nrows_to_infer rows enabling the arguments as specified.
```
Comment (TobyRoseman) [inline src/python/turicreate/data_structures/sframe.py]: I don't really understand this description. I would just say something like: `The number of rows used to infer column types.`

**Comment by tapaswenipathak** [comment]
@TobyRoseman: review ping.

**Comment by TobyRoseman** [comment]
@tapaswenipathak - thanks for the ping. Also thanks for the helpful pull request and addressing my feedback.

**Pull request merged**